samples/lazy_async_modals_app.py

```diff
@@ -125,24 +135,27 @@ async def show_multi_options(ack):
 
 
 @app.view("view-id")
-async def view_submission(ack, payload, logger):
+async def handle_view_submission(ack, payload, logger):
     await ack()
     logger.info(payload["view"]["state"]["values"])
 
 
-@app.action("a")
-async def button_click(ack, respond):
+async def ack_button_click(ack, respond):
     await ack()
+    await respond("Loading ...")
+
+
+async def respond_5_seconds_later(respond):
```

Comment:
You don't need to (and cannot) call `ack()` method here as it's a lazy listener function.